devel/testnet/README.md

```diff
@@ -143,7 +143,7 @@ points to your own IP address.
 As you have read in the previous section, you can run a validator, but if the validator runs on
 a local computer without a publicly exposed ip, it cannot be reached from other validators.
 One option is to use a service like [ngrok](https://ngrok.com) to expose a local port to the public. If you only run
-a fullnode, not a validator, please skip this section.
+a full node, not a validator, please skip this section.
```

Comment:
I think `fullnode` was correct here.

Reply:
Out of curiosity, can you explain the difference between them?

Reply:
Looking at the rest of the `Readme` it does use `full node` and not `fullnode` (the name we use for the class in the code) so let's keep it consistent.